ci/keycloak/keycloak_functions.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+KEYCLOAK_SERVICE_NAME="oidc-keycloak"
```

Comment:
@nessiLahav did you move the scripts so that it is not oidc specific and can be used for jwt? if so - can you please remove any occurrence of `oidc` from these files?

Also - i think the scripts should be at least under an `oauth` directory to make some sense. then both oidc and jwt can use them.

Reply:
@orenbm Do you think i should change also the container name from `oidc-keycloak` to `keycloak` ?

Reply:
Done please review